CHANGES.rst

```diff
@@ -1,5 +1,9 @@
 Changes
 -------
+2.3.3 (2022-05-09)
+^^^^^^^^^^^^^^^^^^
+* fix Test files appearing in distribution package
```

Comment:
need to bump version in __init__.py as well

Reply:
versions bumped